src/views/components/HeroSelector/HeroSelector.js

```diff
@@ -43,20 +36,20 @@ export class HeroSelector extends ComponentView {
             this.playerForm.on('setAvatarButtonState-Passive', () => this.setButtonPassive(heroAvatarButton));
             this.playerForm.on(`setAvatarButtonState-Active-${hero.id}`, () => this.setButtonActive(heroAvatarButton));
 
-            this.avatarButtonsContainer.appendChild(heroAvatarButton);
+            this.el.appendChild(heroAvatarButton);
         });
     }
 
     setButtonPassive(button) {
         const buttonElem = button;
 
-        buttonElem.className = 'hero-avatar-button';
+        buttonElem.classList = 'hero-avatar-button';
     }
 
     setButtonActive(button) {
         const buttonElem = button;
 
-        buttonElem.className = 'hero-avatar-button-active';
+        buttonElem.classList = 'hero-avatar-button-active';
```

Comment:
```suggestion
        button.classList.remove('hero-avatar-button-active');
```
також класік можна простіше зробити, наприклад, просто `active`